securedrop_client/crypto.py

```diff
@@ -37,8 +39,39 @@ class CryptoError(Exception):
     pass
 
 
-class GpgHelper:
+def read_gzip_header_filename(filename: str) -> str:
+    """
+    Extract the original filename from the header of a gzipped file.
+
+    Adapted from Python's gzip._GzipReader._read_gzip_header.
+    """
+    original_filename = ""
+    with open(filename, "rb") as f:
+        magic = f.read(2)
```

Comment:
instead of "magic" you could say "gzip format magic header"

Reply:
Sure. Would you like me to do the same for the other variables representing header components (method, flag, mtime)?

Reply:
sounds good to me, it would be clearer to have a list of our magic values together, i think

Reply:
but I like how concise this code is!